### DISCUSSION TITLE
Integration with Microsoft AD

### DISCUSSION THREAD

**Discussion by old-syniex** [docker-mailserver/docker-mailserver#2445] [Q&A]
Did anyone succeed to integrate mailserver with Microsoft AD?
I tried to find resources about what should be configured in the environments and did not understand what is needed.

Any help will be appreciate.

**Comment by casperklein** [comment]
There is no AD specific documentation, but for LDAP: https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/auth-ldap/
